### ISSUE TITLE
Current errors or red lines during the first `vagrant up`

### ISSUE THREAD

**Issue by jeremyfelt** [Varying-Vagrant-Vagrants/VVV#229]
During provisioning on the first `vagrant up`, there are quite a few areas where red text floats by. These are often not errors, though sometimes are. It can be somewhat jarring and it would be nice to avoid as many as possible.

I'm including a checklist of all the errors currently being shown on a new clone of the repo as of 5c1033dd45d7b179aa3a9196e0feab0443af7e44. Work can be done on this over the next couple releases if needed.

* [ ] `stdin: is not a tty` right when provisioning begins.
* [x] gpg related errors when signing new apt sources
    * `gpg: new configuration file /root/.gnupg/gpg.conf' created`
    * `gpg: WARNING: options in /root/.gnupg/gpg.conf' are not yet active during this run`
    * `gpg: requesting key 7BD9BF62 from hkp server keyserver.ubuntu.com`
    * `gpg: requesting key A2F4C039 from hkp server keyserver.ubuntu.com`
    * `gpg: requesting key E5267A6C from hkp server keyserver.ubuntu.com`
    * `gpg: requesting key E1DF1F24 from hkp server keyserver.ubuntu.com`
    * `gpg: requesting key C7917B12 from hkp server keyserver.ubuntu.com`
* [ ] `dpkg-preconfigure: unable to re-open stdin: No such file or directory` after initial apt packages are downloaded and installation is about to begin.
* [ ] mysql configuration related errors
    * `[Warning] Using unique option prefix key_buffer instead of key_buffer_size is deprecated and will be removed in a future release. Please use the full name instead.`
    * `[Warning] Setting lower_case_table_names=2 because file system for /var/lib/mysql/ is case insensitive
mysql start/running, process 11658`
* [ ] `Creating config file /etc/php5/mods-available/pdo.ini with new version`
* [ ] `Creating config file /etc/php5/fpm/php.ini with new version`
* [ ] `Creating config file /etc/php5/cli/php.ini with new version`
* [ ] `Creating config file /etc/php5/mods-available/gd.ini with new version`
* [ ] `Creating config file /etc/php5/mods-available/imap.ini with new version`
* [ ] `Creating config file /etc/php5/mods-available/mcrypt.ini with new version`
* [ ] `Creating config file /etc/php5/mods-available/mysql.ini with new version`
* [ ] `Creating config file /etc/php5/mods-available/mysqli.ini with new version`
* [ ] `Creating config file /etc/php5/mods-available/pdo_mysql.ini with new version`
* [ ] `Creating config file /etc/php5/mods-available/readline.ini with new version`
* [ ] `Creating config file /etc/papersize with new version`
* [ ] `Creating config file /etc/php5/conf.d/imagick.ini with new version`
* [ ] A large chunk of postfix related noise - [gist](https://gist.github.com/jeremyfelt/7904654)
* [ ] `/tmp/vagrant-shell: line 194: composer: command not found`
* [ ] `/tmp/vagrant-shell: line 244: grunt: command not found`
* [x] `unlink: cannot unlink ...` - *these should be removed in v1.1 as the transition to copying files will be complete in v1.0
* [ ] `Cloning into '/srv/www/wp-cli'...`
* [ ] `Cloning into '/srv/www/default/webgrind'...`
* [ ] `Cloning into '/srv/www/phpcs'... Checking out files: 100% (851/851), done.`
* [ ] `Cloning into '/srv/www/phpcs/CodeSniffer/Standards/WordPress'...`
* [ ] Output from WordPress stable download

**Comment by artemkaint** [comment]
fix for first issue:  adding line Vagrantfile
<pre>
<code>config.ssh.shell = "bash -c 'BASH_ENV=/etc/profile exec bash'"</code>
</pre>
from here https://github.com/mitchellh/vagrant/issues/1673#issuecomment-28288042

**Comment by cfoellmann** [comment]
@jeremyfelt is `Creating config file XXXXXX` an error we need to fix? It is just a notice what is going on, right?

**Comment by cfoellmann** [comment]
@artemkaint I added your suggested fix here https://github.com/cfoellmann/VVV/tree/issue/229-stdin on the `ìssue/229-stdin` branch. It fixes the `stdin: is not a tty` output on provision but it has some negative side effects (at least for me)
I'm running Win8.1 + Vagrant 1.6.3 (I hate updating Vagrant because of the issues all the time) + VB 4.3.20

If I set the fix I can provision without problems __BUT__ if I try to `vagrant destroy` or `vagrant halt` these commands are broken. I don't know why (yet) or if there are other issues.
Maybe this should be moved to its own issue?
